skimage/feature/tests/test_blob.py

```diff
@@ -155,19 +155,76 @@ def test_blob_dog_excl_border():
         max_sigma=5,
         sigma_ratio=1.2,
     )
-    assert blobs.shape[0] == 1
+    assert blobs.shape[0] == 1, "one blob should have been detected"
     b = blobs[0]
     assert b[0] == b[1] == 5, "blob should be 5 px from x and y borders"
-
+    
+    exclude_border_dist = 6
     blobs = blob_dog(
         img,
         min_sigma=1.5,
         max_sigma=5,
         sigma_ratio=1.2,
-        exclude_border=6,
+        exclude_border=exclude_border_dist,
     )
     msg = "zero blobs should be detected, as only blob is 5 px from border"
     assert blobs.shape[0] == 0, msg
+    
+    # verify that tuple exclude_border work
+    blobs = blob_dog(
+        img,
+        min_sigma=1.5,
+        max_sigma=5,
+        sigma_ratio=1.2,
+        exclude_border=(exclude_border_dist,exclude_border_dist)
+    )
+
+    msg = "zero blobs should be detected, as only blob is 5 px from border"
+    assert blobs.shape[0] == 0, msg
```

Comment:
I think this new snippet and the following new test `test_blob_dog_excl_border_multidim` are a bit redundant. I'd rather keep tests as short and contained as possible. How about consolidating this with `pytest.mark.parametrize` like this:
```python
@pytest.mark.parametrize("exclude_border", [6, (6, 6)])
def test_blob_dog_excl_border(exclude_border):
    # Testing exclude border
    # image where blob is 5 px from borders, radius 5
    img = np.ones((512, 512))
    xs, ys = disk((5, 5), 5)
    img[xs, ys] = 255
    # [...]
    blobs = blob_dog(
        img,
        min_sigma=1.5,
        max_sigma=5,
        sigma_ratio=1.2,
        exclude_border=exclude_border,
    )
    # [...]
```
Could you do the same for `test_blob_log_exclude_border` so that function is covered as well?

Reply:
yeah, that is a good idea, I'm not very proficient in tests